gnomad/utils/annotations.py

```diff
@@ -1237,6 +1226,7 @@ def get_gks(
     :return: Dictionary containing VRS information (and frequency information split by ancestry groups and sex if desired) for the specified variant.
 
     """
+    from gnomad.utils.filtering import get_reference_genome
```

Comment:
should be at top of script

Reply:
should also be this instead right?

`from gnomad.utils.reference_genome import get_reference_genome`

Reply:
yep! I moved it down here to fix an odd import error, where 'filtering' was referencing a method in 'annotations' that hadn't been defined yet, but changing it to Julia's suggestion fixes the issue and lets us move the import to the top of the script to be in-line with Methods style. thanks team